src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -8,27 +8,56 @@
 
 import com.revrobotics.spark.SparkLowLevel.MotorType;
 import com.revrobotics.spark.SparkMax;
+
+import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.MotorIDConstants;
+import com.ctre.phoenix6.hardware.TalonFX;
+// import 
 
 public class IntakeSubsystem extends SubsystemBase {
   /** Creates a new IntakeSubsystem. */
   private SparkMax intakeMotor;
-
+  private TalonFX pivotMotor;
+  private SparkMax conveyorMotor;
+  private Encoder pivotEncoder;
   public IntakeSubsystem() {
-    intakeMotor = new SparkMax(MotorIDConstants.kIntakeMotor, MotorType.kBrushed);
+    intakeMotor = new SparkMax(MotorIDConstants.kIntakeMotor, MotorType.kBrushless);
+    pivotMotor = new TalonFX(MotorIDConstants.kPivotMotor);
+    conveyorMotor = new SparkMax(MotorIDConstants.kConveyorMotor, MotorType.kBrushless);
+    pivotEncoder = new Encoder(0, 1);
   }
-
+  public Command setPivotMotor(double speed) {
```

Comment:
This won't work well.  As above, you will want a positional PID for this motor

Reply:
again, this would be usful for testing, but we will want to move away from it, once we know it works